sklearn/preprocessing/tests/test_common.py

```diff
@@ -27,7 +29,8 @@ def _get_valid_samples_by_column(X, col):
 
 @pytest.mark.parametrize(
     "est, func, support_sparse",
-    [(MinMaxScaler(), minmax_scale, False),
+    [(MaxAbsScaler(), maxabs_scale, True),
+     (MinMaxScaler(), minmax_scale, False),
      (QuantileTransformer(n_quantiles=10), quantile_transform, True)]
 )
 def test_missing_value_handling(est, func, support_sparse):
```

Comment:
I've realised we should have assert_no_warnings in here when fitting and transforming

Reply:
It is a good point. I just catched that the QuantileTransformer was still raising a warning at inverse_transform. Since it is a single line I would included here.